src/components/shared/thread/ActivityGraph.js

```diff
@@ -130,12 +136,11 @@ export class ThreadActivityGraph extends React.PureComponent<Props, State> {
     }
     // Re-measure the canvas and get the coordinates and time for the click.
     const { rangeStart, rangeEnd } = this.props;
-    const rect = canvas.getBoundingClientRect();
-    const x = event.pageX - rect.left;
-    const y = event.pageY - rect.top;
-    const time = rangeStart + (x / rect.width) * (rangeEnd - rangeStart);
+    const x = event.nativeEvent.offsetX;
+    const y = event.nativeEvent.offsetY;
```

Comment:
I also use directly `event.offsetX/Y` as they're the right tool for the job here. Therefore you'll see some changes about that in the tests.